### PR TITLE
Update learn/manage/actions docs for dashboard v15

### PR DESCRIPTION
**Pull request by myarmolinsky**
Update learn/manage/actions docs for dashboard v15

Change-type: minor
Signed-off-by: Matthew Yarmolinsky <matthew-timothy@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
As a band-aid this will work, I will start working on #2333 ASAP

File: pages/learn/manage/actions.md [hunk 1]
```diff
@@ -68,83 +54,65 @@ The `Shutdown` action allows you to safely shut down your devices. It should be
...
-## Device-specific actions
+### Delete Device
+
+The `Delete Device` action is an extremely dangerous action and results in disassociating the device from the fleet and remote endpoint. Once you have deleted a device from the fleet it is not possible to reconnect to it unless you set it back up again. The device itself will continue to run the container and code you pushed most recently, but will never be able to receive new updates or commands from the {{ $names.company.lower }} dashboard or API.
+
+## Device settings
 
 [Fleet members][fleet-members] with the Operator role and above can perform any of the actions listed below.
 
-### Update Locking
```
[the remark this answers is not in the document]
Comment (myarmolinsky) [reply pages/learn/manage/actions.md:75]: Hasn't been available for a bit now. It used to be in the actions dropdown, but we decided to leave it as just a config variable. This was pre-v15
image

**Comment by myarmolinsky** [comment]
> As a band-aid this will work, I will start working on #2333 ASAP

Cool, should I self-certify then? Or are we holding for some reason?

File: pages/learn/manage/actions.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/actions.md:75]: Cool beans

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**